lldb/test/API/lang/swift/variables/move_function/TestSwiftMoveFunction.py

```diff
@@ -0,0 +1,213 @@
+        # ccf is conditional control flow
+        self.do_check_copyable_value_ccf_true()
+        self.do_check_copyable_value_ccf_false()
+
+    def setUp(self):
+        TestBase.setUp(self)
+        self.main_source = "main.swift"
+        self.main_source_spec = lldb.SBFileSpec(self.main_source)
+        self.exec_name = "a.out"
+
+    def add_breakpoints(self, name, num_breakpoints):
+        pattern = 'Set breakpoint {} here {}'
+        for i in range(num_breakpoints):
+            pat = pattern.format(name, i+1)
+            brk = self.target.BreakpointCreateBySourceRegex(
+                pat, self.main_source_spec)
+            self.assertTrue(brk.GetNumLocations() > 0, VALID_BREAKPOINT)
+            yield brk
+
+    def do_setup_breakpoints(self):
+        self.breakpoints = []
+
+        self.breakpoints.extend(
+            self.add_breakpoints('copyableValueTest', 3))
+        self.breakpoints.extend(
+            self.add_breakpoints('addressOnlyValueTest', 3))
+        self.breakpoints.extend(
+            self.add_breakpoints('copyableVarTest', 4))
+        self.breakpoints.extend(
+            self.add_breakpoints('addressOnlyVarTest', 4))
+        self.breakpoints.extend(
+            self.add_breakpoints('copyableValueCCFTrueTest',
+                                 5))
+        self.breakpoints.extend(
+            self.add_breakpoints('copyableValueCCFFalseTest',
+                                 3))
+
+    def do_check_copyable_value_test(self):
+        frame = self.thread.frames[0]
+        self.assertTrue(frame.IsValid(), "Couldn't get a frame.")
+
+        # We haven't defined varK yet.
+        varK = frame.FindVariable('k')
+
+        self.assertTrue(varK.value == None, "varK initialized too early?!")
+
+        # Go to break point 2. k should be valid.
+        self.runCmd('continue')
+        self.assertTrue(varK.unsigned > 0, "varK not initialized?!")
+
+        # Go to breakpoint 3. k should no longer be valid.
+        self.runCmd('continue')
+        self.assertTrue(varK.value == None, "K is live but was moved?!")
+
+        # Run so we hit the next breakpoint to jump to the next test's
+        # breakpoint.
+        self.runCmd('continue')
+
+    def do_check_copyable_var_test(self):
+        frame = self.thread.frames[0]
+        self.assertTrue(frame.IsValid(), "Couldn't get a frame.")
+
+        # We haven't defined varK yet.
+        varK = frame.FindVariable('k')
+        self.assertTrue(varK.value == None, "varK initialized too early?!")
+
+        # Go to break point 2. k should be valid.
+        self.runCmd('continue')
+        self.assertTrue(varK.unsigned > 0, "varK not initialized?!")
+
+        # Go to breakpoint 3. We invalidated k
+        self.runCmd('continue')
+        self.assertTrue(varK.value == None, "K is live but was moved?!")
+
+        # Go to the last breakpoint and make sure that k is reinitialized
+        # properly.
+        self.runCmd('continue')
```

Comment:
if you store the process above, you could do `process.Continue()` here, which is slightly nicer than parsing a string command.